config/initializers/00_rest_client.rb

```diff
@@ -0,0 +1,7 @@
+require 'rest_client'
```

Comment:
left over? or maybe this should be redirected to its own logger (i would say thats a feature by it self).